README.md

```diff
@@ -338,6 +338,16 @@ MIDDLEWARE = [
     # ... other middlewares
 ]
 ```
+
+You can configure the X-Ray recorder in a Django app under the ‘XRAY_RECORDER’ namespace. For a minimal configuration, the 'AWS_XRAY_TRACING_NAME' is required unless it is specified in an environment variable. 
+```
+XRAY_RECORDER = {
+    'AWS_XRAY_TRACING_NAME': 'My application', # Required - the segment name for segments generated from incoming requests
+}
+```
+For more information about configuring Django with X-Ray read more about it in the API reference:
```

Comment:
Instead of adding separate URL, lets follow markdown style of hyperlink.

Example [API reference](https://docs.aws.amazon.com/xray-sdk-for-python/latest/reference/frameworks.html)

Reply:
Done :)